pod/perlgit.pod

```diff
@@ -814,35 +814,6 @@ original commit in the new commit message.
 Before pushing any change to a maint version, make sure you've
 satisfied the steps in L</Committing to blead> above.
 
-=head2 Merging from a branch via GitHub
```

Comment:
This is cool but we probably need a TODO to add something about PRs in github

Reply:
I would like to do this as a separate PR, the references to perlbug are extensive.